src/components/input/style.ts

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+import { Input } from "./model";
+
+export const InputContainer = styled.input<Input>`
+  width: ${props => props.width ? props.width : "150px"};
+  font-size: 25px;
+  background: none;
+  outline: none;
```

Comment:
outline is globally set to 'none'; you should be able to delete this line without any side effects